src/plugins/profiling/server/routes/flamegraph.ts

```diff
@@ -72,7 +73,7 @@ export class FlameGraph {
   totalCount: number;
 
   events: any;
```

Comment:
Now that we have an explicit type for stack trace events, we could use `Array<StackTraceEvent>` here.